modules/nw-ingress-route-secret-load-external-cert.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nnw-ingress-route-secret-load-external-cert_{context}"]
+= Route Secret load For External Certificate
+
+Currently, users of OpenShift cannot easily integrate OpenShift Routes with third-party certificate management solutions like cert-manager. The existing design of the Routes API mandates that certificate data be embedded within the Route object, rather than referenced. This creates challenges for seamless integration and automated lifecycle management.
+
```

Comment:
Do we need to talk about the previous challenge here? I'm not sure about the standard practice.

Reply:
Will be deleted.